server/sonar-server/src/main/java/org/sonar/server/user/ws/CreateAction.java

```diff
@@ -20,6 +20,8 @@
 package org.sonar.server.user.ws;
```

Comment:
you haven't added a test for `CreateActionTest` and `UpdateActionTest` to cover the use case you're solving, which means several scm accounts, with one of the users having a comma in its scm account